recipe/meta.yaml

```diff
@@ -38,7 +38,7 @@ requirements:
         - numpy
         - ocl-icd  # [linux]
         - khronos-opencl-icd-loader  # [win]
-        - mingwpy  # [win]
+        - mingwpy  # [win and (py27 or py34 or py35)]
```

Comment:
Thanks for checking! I'm not too fond of that because it'll promptly break at `py37`.